web/client/components/map/openlayers/plugins/WFS3Layer.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright 2018, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import head from 'lodash/head';
+import urlParser from 'url';
+
+import CoordinatesUtils from '../../../../utils/CoordinatesUtils';
+import MapUtils from '../../../../utils/MapUtils';
+import Layers from '../../../../utils/openlayers/Layers';
+import SecurityUtils from '../../../../utils/SecurityUtils';
+
+import {get, getTransform} from 'ol/proj';
+import {applyTransform} from 'ol/extent';
+import TileGrid from 'ol/tilegrid/TileGrid';
+import VectorTileLayer from 'ol/layer/VectorTile';
+import VectorTile from 'ol/source/VectorTile';
+import MVT from 'ol/format/MVT';
+
+import { isVectorFormat } from '../../../../utils/VectorTileUtils';
+import { getStyle } from '../VectorStyle';
+import { OL_VECTOR_FORMATS } from '../../../../utils/openlayers/VectorTileUtils';
+
+const applyStyle = (vectorStyle, layer) => {
```

Comment:
Can we put this in openlayers VectorTileUtils since we have the same in WMSLayer?

Reply:
@mbarto updated